packages/iris-grid/src/sidebar/visibility-ordering-builder/VisibilityOrderingGroup.tsx

```diff
@@ -171,43 +205,47 @@ export default function VisibilityOrderingGroup(
             )
           }
           tooltip="Set color"
-          onClick={() => {
-            colorInputRef.current?.click();
-            // Mostly for testing. Chrome seems to not give the hidden input focus
-            // Really would only affect screen readers
-            colorInputRef.current?.focus();
-          }}
+          /**
+           * Toggle to close the picker on Chrome
+           * Prevents Firefox on Mac from getting into a stuck state
+           * Does not close on Firefox b/c the picker stays open when the element is removed
+           */
+          onClick={() => setIsColorInputOpen(val => !val)}
```

Comment:
This is annoying on Chrome - if I click the colour picker, select a colour, the colour picker disappears... and then if I click the colour dropper to show it again I need to click twice. Not sure if there's a better way.

Reply:
Just changed it to click instead of focus. Try that.

With focus, clicking the button made it close on chrome. The behavior prior to this change was clicking always opened on chrome, so this shouldn't change any chrome behavior now. There's no consistent event when a color input is closed (no blur event)